source/extensions/network/dns_resolver/getaddrinfo/getaddrinfo.cc

```diff
@@ -148,6 +150,7 @@ void GetAddrInfoDnsResolver::resolveThreadRoutine() {
     std::pair<ResolutionStatus, std::list<DnsResponse>> response;
     std::string details;
     {
+      next_query->addDetail("starting");
```

Comment:
Coming back to this, if we think it could end up in trace logs, I like having the longer strings. If we think it'll only ever go in the response code detail field for Envoy Mobile, I wonder if we want to do a character-per-operation because of log limits, unless we sorted that issue out